packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -294,58 +275,77 @@ describe('Checkout', () => {
-                authToken: 'mytoken'
-            };
+        it('should call `initialise`', () => {
+            // Arrange
+            const initialiseSpy = jest.spyOn(VueCheckout.methods, 'initialise');
 
+            // Act
             shallowMount(VueCheckout, {
                 store: createStore(),
                 i18n,
                 localVue,
-                propsData: propsDataWithAuthToken
+                propsData
             });
 
-            expect(setAuthTokenSpy).toHaveBeenCalledWith(propsDataWithAuthToken.authToken);
+            // Assert
+            expect(initialiseSpy).toHaveBeenCalled();
         });
+    });
 
-        it('should call `loadCheckout`', () => {
-            // Arrange & Act
-            const loadCheckoutSpy = jest.spyOn(VueCheckout.methods, 'loadCheckout');
+    describe('methods ::', () => {
+        afterEach(() => {
+            jest.clearAllMocks();
+        });
 
-            shallowMount(VueCheckout, {
-                store: createStore(),
-                i18n,
-                localVue,
-                propsData
+        describe('initialise ::', () => {
+            it('should call `setAuthToken`', () => {
+                // Arrange & Act
+                const setAuthTokenSpy = jest.spyOn(VueCheckout.methods, 'setAuthToken');
+
+                const propsDataWithAuthToken = {
+                    ...propsData,
+                    authToken: 'mytoken'
+                };
+
+                shallowMount(VueCheckout, {
+                    store: createStore(),
+                    i18n,
+                    localVue,
+                    propsData: propsDataWithAuthToken
+                });
+
+                expect(setAuthTokenSpy).toHaveBeenCalledWith(propsDataWithAuthToken.authToken);
             });
 
-            expect(loadCheckoutSpy).toHaveBeenCalled();
-        });
+            it('should call `loadCheckout`', () => {
+                // Arrange & Act
+                const loadCheckoutSpy = jest.spyOn(VueCheckout.methods, 'loadCheckout');
 
-        it('should call `loadAvailableFulfilment`', () => {
-            // Arrange & Act
-            const loadAvailableFulfilmentSpy = jest.spyOn(VueCheckout.methods, 'loadAvailableFulfilment');
+                shallowMount(VueCheckout, {
+                    store: createStore(),
+                    i18n,
+                    localVue,
+                    propsData
+                });
 
-            shallowMount(VueCheckout, {
-                store: createStore(),
-                i18n,
-                localVue,
-                propsData
+                expect(loadCheckoutSpy).toHaveBeenCalled();
             });
 
-            expect(loadAvailableFulfilmentSpy).toHaveBeenCalled();
-        });
-    });
+            it('should call `loadAvailableFulfilment`', () => {
+                // Arrange & Act
+                const loadAvailableFulfilmentSpy = jest.spyOn(VueCheckout.methods, 'loadAvailableFulfilment');
```

Comment:
Minor one but I think it's better to set the spy up after we've mounted the component. I haven't checked but i'm pretty sure this test (& others would too) if the methods object was removed from the instance or ported to say a mixin or computed prop? With the wrapper instance it would fail within the test if the method was removed completely from the vm instance and would be a better failure imo.

Reply:
Hmmm, haven't seen that behaviour, @kevinrodrigues. You may be right, though.
All our tests in checkout are structured this way. For now, are you ok with leaving it as it is?